jdbc/src/main/java/software/amazon/timestream/jdbc/TimestreamDatabaseMetaData.java

```diff
@@ -935,6 +935,10 @@ public boolean supportsSavepoints() {
     return false;
   }
 
+  /**
+   * Retrieves whether a schema name can be used in a data manipulation statement.
```

Comment:
Timestream JDBC driver 1.0 already supports "SELECT" DML command ( Example query:  SELECT col1,col2 FROM database1.table1 ) 

Can you help us understand with an example why "supportsSchemasInDataManipulation= true" needs to be set as part of this commit?

Reply:
It is needed as for some BI tools(DBeaver, SQuirreL) need it to generate/display correct schemas. For DBeaver, if supportsSchemasInDataManipulation is false, the generated query after you double click the table is “select * from table”. As Timestream requires database be prefixed to table, the generated query could not run. After supportsSchemasInDataManipulation is set to true, the generated query is “select * from database.table” which could run successfully. For SQuirreL, when supportsSchemasInDataManipulation is false, the table display will not have schema layer. That is, table is displayed under connection name directly. For duplicated tables in different databases, there is only one table displayed. After supportsSchemasInDataManipulation is set to true, under the connection name there is a layer of schema. Tables are displayed under schemas. That is the schemas/tables could be displayed correctly. You can also refer to Squirrel codebase as an example where this parameter is used - [squirrel-sql-code/DatabaseObjectInfo.java at dd3f19b06f426fb601cd25ea3150ca3f3c5b99a3 · squirrel-sql-client/squirrel-sql-code](https://github.com/squirrel-sql-client/squirrel-sql-code/blob/dd3f19b06f426fb601cd25ea3150ca3f3c5b99a3/sql12/core/src/net/sourceforge/squirrel_sql/fw/sql/DatabaseObjectInfo.java#L272)